scripts/setup-adapters.sh

```diff
@@ -23,6 +23,111 @@ SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
+      cmake --build cmake-out -- -j ${NCPU:-4} && \
+  $sudocmd cmake --build cmake-out --target install -- -j ${NCPU:-4} && \
+  $sudocmd ldconfig
+
+
+  # install protobuf
+  echo " Installing protobuf..."
+  mkdir -p ${DEPENDENCY_DIR}/protobuf && cd ${DEPENDENCY_DIR}/protobuf
+  curl -sSL https://github.com/protocolbuffers/protobuf/archive/v3.20.1.tar.gz | \
+      tar -xzf - --strip-components=1 && \
+      cmake \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DBUILD_SHARED_LIBS=yes \
+          -Dprotobuf_BUILD_TESTS=OFF \
+          -Hcmake -Bcmake-out && \
+      cmake --build cmake-out -- -j ${NCPU:-4} && \
+  $sudocmd cmake --build cmake-out --target install -- -j ${NCPU:-4} && \
+  $sudocmd ldconfig
+
+  # install gRPC
+  echo " Installing gRPC..."
+  mkdir -p ${DEPENDENCY_DIR}/grpc && cd ${DEPENDENCY_DIR}/grpc
+  curl -sSL https://github.com/grpc/grpc/archive/v1.45.2.tar.gz | \
+      tar -xzf - --strip-components=1 && \
+      cmake \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DgRPC_INSTALL=ON \
+          -DgRPC_BUILD_TESTS=OFF \
+          -DgRPC_ABSL_PROVIDER=package \
+          -DgRPC_CARES_PROVIDER=package \
+          -DgRPC_PROTOBUF_PROVIDER=package \
+          -DgRPC_RE2_PROVIDER=package \
+          -DgRPC_SSL_PROVIDER=package \
+          -DgRPC_ZLIB_PROVIDER=package \
+          -H. -Bcmake-out && \
+      cmake --build cmake-out -- -j ${NCPU:-4} && \
+  $sudocmd cmake --build cmake-out --target install -- -j ${NCPU:-4} && \
+  $sudocmd ldconfig
+
+  # install crc32c
+  echo " Installing crc32c..."
+  mkdir -p ${DEPENDENCY_DIR}/crc32c && cd ${DEPENDENCY_DIR}/crc32c
+  curl -sSL https://github.com/google/crc32c/archive/1.1.2.tar.gz | \
+      tar -xzf - --strip-components=1 && \
+      cmake \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DBUILD_SHARED_LIBS=yes \
+          -DCRC32C_BUILD_TESTS=OFF \
+          -DCRC32C_BUILD_BENCHMARKS=OFF \
+          -DCRC32C_USE_GLOG=OFF \
+          -H. -Bcmake-out && \
+      cmake --build cmake-out -- -j ${NCPU:-4} && \
+  $sudocmd cmake --build cmake-out --target install -- -j ${NCPU:-4} && \
+  $sudocmd ldconfig
+
+  # install nlohmann_json library
+  echo " Installing nlohmann json..."
+  mkdir -p ${DEPENDENCY_DIR}/json && cd ${DEPENDENCY_DIR}/json
+  curl -sSL https://github.com/nlohmann/json/archive/v3.10.5.tar.gz | \
+      tar -xzf - --strip-components=1 && \
+      cmake \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DBUILD_SHARED_LIBS=yes \
+        -DBUILD_TESTING=OFF \
+        -DJSON_BuildTests=OFF \
+        -H. -Bcmake-out/nlohmann/json && \
+  $sudocmd cmake --build cmake-out/nlohmann/json --target install -- -j ${NCPU:-4} && \
+  $sudocmd ldconfig
+  echo " Clone, compile and install google-cloud-cpp..."
+  $sudocmd rm -rf ${DEPENDENCY_DIR}/google-cloud-cpp
+  # clone and compile the main project
+  git clone https://github.com/googleapis/google-cloud-cpp.git ${DEPENDENCY_DIR}/google-cloud-cpp
+  cd ${DEPENDENCY_DIR}/google-cloud-cpp
```

Comment:
So in circle you have been using `~/adapter-deps/google-cloud-cpp-installed` and here the path is different?